.github/workflows/downstream_ci.yml

```diff
@@ -32,7 +29,14 @@ jobs:
           ruby-version: ${{ env.ruby_version }}
           bundler-cache: true
 
-      - name: publish smithy-ruby to maven local
+      - uses: actions/checkout@v4
+
+      - name: Install gems
```

Comment:
I don't think we need to do this here, should be handled by using checkout and then setup ruby after.

Reply:
It was needed to get CI to pass. Try removing it and see.